src/misc/lv_text.c

```diff
@@ -649,6 +649,13 @@ static uint32_t lv_text_utf8_next(const char * txt, uint32_t * i)
     uint32_t i_tmp = 0;
     if(i == NULL) i = &i_tmp;
 
+    /* Ensure the string is not null */
+    if(txt == NULL || *i >= lv_strlen(txt)) {
+        LV_LOG_WARN("txt is NULL or empty");
```

Comment:
I saw lot of these log messages, because it's valid that the next letter in `lv_text_encoded_letter_next_2` is `\0` when we are the last but one letter. In light of that I think this log should be removed.

Reply:
Removed the log.